internal/app/connectconformance/connectconformance.go

```diff
@@ -281,22 +281,26 @@ func run( //nolint:gocyclo
 		clients = []processInfo{
 			{
 				name: "reference client",
-				start: runInProcess([]string{
-					"reference-client",
-					"-p", strconv.Itoa(int(flags.Parallelism)),
-				}, func(ctx context.Context, args []string, inReader io.ReadCloser, outWriter, errWriter io.WriteCloser) error {
-					return referenceclient.RunInReferenceMode(ctx, args, inReader, outWriter, errWriter, trace)
-				}),
+				start: runInProcess(
+					[]string{
+						"reference-client",
+						"-p", strconv.Itoa(int(flags.Parallelism)),
+					}, func(ctx context.Context, args []string, inReader io.ReadCloser, outWriter, errWriter io.WriteCloser) error {
+						return referenceclient.RunInReferenceMode(ctx, args, inReader, outWriter, errWriter, trace)
+					},
+				),
```

Comment:
Sorry if this is too noisy. I tweaked the formatting of this block and similar ones below which made it easier for me to read.